Microsoft.Toolkit.Uwp.UI.Media/Helpers/Win2DImageHelper.cs

```diff
@@ -0,0 +1,297 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Numerics;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Microsoft.Graphics.Canvas;
+using Microsoft.Graphics.Canvas.UI;
+using Microsoft.Graphics.Canvas.UI.Composition;
+using Microsoft.Graphics.Canvas.UI.Xaml;
+using Microsoft.Toolkit.Uwp.UI.Media.Extensions.System;
+using Microsoft.Toolkit.Uwp.UI.Media.Extensions.System.Threading.Tasks;
+using Microsoft.Toolkit.Uwp.UI.Media.Helpers.Cache;
+using Windows.Foundation;
+using Windows.Graphics.DirectX;
+using Windows.Graphics.Display;
+using Windows.Graphics.Imaging;
+using Windows.UI;
+using Windows.UI.Composition;
+using Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Uwp.UI.Media.Helpers
+{
+    /// <summary>
+    /// A helper <see langword="class"/> that loads Win2D images and manages an internal cache of <see cref="CompositionBrush"/> instances with the loaded images
+    /// </summary>
+    public static class Win2DImageHelper
```

Comment:
Is this similar at all to our SurfaceLoader class?  Wondering the overlap here.

Reply:
Ah, good catch, yeah there's definitely some overlap.
I'll create a branch and port the `SurfaceLoader` class over, and merge the two 👍

Reply:
Done in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3112/commits/e7d8b36206538e37fbf5303b5e7e4c738845e6c5.